llvm/utils/release/build_llvm_release.bat

```diff
@@ -250,7 +252,7 @@ set cmake_flags=^
   -DLLDB_TEST_COMPILER=%stage0_bin_dir%/clang.exe ^
   -DPYTHON_HOME=%PYTHONHOME% ^
   -DPython3_ROOT_DIR=%PYTHONHOME% ^
-  -DLIBXML2_INCLUDE_DIRS=%libxmldir%/include/libxml2 ^
+  -DLIBXML2_INCLUDE_DIR=%libxmldir%/include/libxml2 ^

```

Comment:
This seems unrelated?

Reply:
Removed. We can't build with the latest cmake without this, will propose that in a separate PR.